dbclient/dbclient/src/main/java/io/helidon/dbclient/DbExecute.java

```diff
@@ -354,4 +354,23 @@ default DbStatementDml createNamedDelete(String statementName, String statement)
     default Single<Long> dml(String statement, Object... parameters) {
         return createDmlStatement(statement).params(parameters).execute();
     }
+
+    /*
+     * Unwrap support
+     */
+
+    /**
+     * Unwrap database executor internals.
+     * Only database connection is supported. Any operations based on this connection are <b>blocking</b>.
+     * Reactive support must be implemented in user code. This connection instance is being used to execute
+     * all statements in current database executor context.
+     * Mongo database driver internals support {@code MongoDatabase} class.
```

Comment:
Same comment regarding the impl reference.

Reply:
Removed impl reference.